stream.go

```diff
@@ -81,13 +81,24 @@ func (s *Stream) Remove(c *Client) {
 }
 
 // Broadcast sends the event to all clients registered on this stream.
-func (s *Stream) Broadcast(e *Event) {
+func (s *Stream) Broadcast(e *Event) (broadcastCount int, clientErrs []error) {
```

Comment:
This breaks the `Broadcast` interface and would be a major version bump. I'm also not sure it's too useful, since the list of errors can't be mapped back to particular clients it's hard to take any action on.

If it's important to surface individual client errors, we should maybe find a way to push them to the client objects.